modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_xml.es.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import formatXML from '../../../src/main/resources/META-INF/resources/liferay/util/format_xml.es';
+
+describe(
+	'Liferay.Util.formatXML',
+	() => {
+		it(
+			'should throw error if content parameter is not a string',
+			() => {				
+				expect(() => formatXML({})).toThrow('must be a string');
+			}
+		);
+
+		it(
+			'should return an XML string if content parameter is an XML string',
+			() => {
+				const content = `
+					<?xml xlmns:a="http://www.w3.org/TR/html4/" version="1.0" encoding="UTF-8"?>
```

Comment:
We should put the ugly string first, and formatted string in the second part of comparison, just to make test a bit more readable.